.github/workflows/reusable-zizmor.yml

```diff
@@ -113,22 +113,18 @@ jobs:
           EOC
           echo "zizmor-config=${ZIZMOR_CONFIG}" | tee -a "${GITHUB_OUTPUT}"
 
-      - name: Setup Rust
-        uses: actions-rust-lang/setup-rust-toolchain@9d7e65c320fdb52dcd45ffaa68deb6c02c8754d9 # v1.12.0
-        with:
-          cache-on-failure: true
+      - uses: astral-sh/setup-uv@c7f87aa956e4c323abf06d5dec078e358f6b4d04 # v6.0.0
 
-      - name: Get zizmor
-        shell: sh
+      - name: Create UV venv & install Zizmor
```

Comment:
What do you think about using [`setup-uv`](https://github.com/astral-sh/setup-uv)?

```yaml
- name: Install the latest version of uv
  uses: astral-sh/setup-uv@6b9c6063abd6010835644d4c2e1bef4cf5cd0fca # v6.0.1
  with:
    enable-cache: true
```

I got this from [Zizmor's docs](https://woodruffw.github.io/zizmor/usage/#use-in-github-actions).

Reply:
Updated the setup uv step to enable cache and use `activate-environment: true` to avoid manually creating the venv. Good call!